Podfile.lock

```diff
@@ -1,5 +1,5 @@
 PODS:
-  - Backtrace (1.7.4-beta2):
+  - Backtrace (1.7.5):
```

Comment:
can we add `-beta` to version? We don't want anyone to treat is as official version. I also think we should bump the version to 1.8.0